src/components/App/index.js

```diff
@@ -438,23 +443,25 @@ class App extends React.Component {
       new Date(Math.min(this.state.ranges.donations[0], this.state.ranges.projects[0])),
       new Date(Math.max(this.state.ranges.donations[1], this.state.ranges.projects[1]))
     ];
-
+    const embedClass = this.state.embed ? 'l-app is-embed' : 'l-app';
 
     return (
-      <div className="l-app">
+      <div className={embedClass} >
 
         <div id="map" className="l-map" ref="Map"></div>
 
-        <button className="btn-share btn-primary l-share" onClick={ () => this.handleModal('open', 'shareOpen') }>
-          <svg className="icon icon-share">
-            <use xlinkHref="#icon-share"></use>
-          </svg>
-        </button>
+        { !this.state.embed ?
+          <button className="btn-share btn-primary l-share" onClick={ () => this.handleModal('open', 'shareOpen') }>
+            <svg className="icon icon-share">
+              <use xlinkHref="#icon-share"></use>
+            </svg>
+          </button> : ''}
```

Comment:
Could we use here simplified condition notation ( && ) as per the others items here?